app/cinema/bar.py

```diff
@@ -0,0 +1,8 @@
+from app.people.customer import Customer
+
+
+class CinemaBar:
+
+    @staticmethod
+    def sell_product(customer: Customer, product: str, ):
```

Comment:
Don't leave commas and empty spaces after arguments if you write it inline. 
```suggestion
    def sell_product(customer: Customer, product: str):
```